react/src/components/VFolderTable.tsx

```diff
@@ -145,16 +145,34 @@ const VFolderTable: React.FC<VFolderTableProps> = ({
     }
   }, [aliasMap, internalForm, aliasBasePath]);
 
+  const formInstance = Form.useFormInstance();
+  const ownerInfo = formInstance.getFieldValue(['owner']);
+  // valid when owner is enabled and all fields are not undefined
+  const isValidOwner =
+    ownerInfo?.enabled &&
+    (!_.isEmpty(ownerInfo)
+      ? _.every(_.omit(ownerInfo, 'enabled'), (key, value) => {
+          return key !== undefined;
+        })
+      : false);
+  const ownerEmail = isValidOwner ? ownerInfo.email : undefined;
+
+  useEffect(() => {
+    formInstance.setFieldsValue({ mounts: undefined, vfolderAliasMap: {} });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [isValidOwner]);
+
```

Comment:
VFolderTable is not a Form.Item component. Therefore, it can be used outside of a Form. If you want to impose a condition based on a specific user's email, VFolderTable should receive the email as a prop. The email condition is an input of this Component.

Reply:
Modified the `VFolderTableFormItem` that wraps the VFolderTable in a Form.Item component to be responsible for getting the value for the email and validating it.